src/app/pages/pass-finalisation/cat-adi-part3/components/reason-given/reason-given.component.html

```diff
@@ -0,0 +1,33 @@
+<ion-row class="mes-validated-row mes-narrow-row" id="further-development-card"
+         [formGroup]="formGroup">
+<div [class.invalid-bar]="!formGroup?.get('reasonGiven').valid && formGroup?.get('reasonGiven').dirty" class="padding-top-22">
```

Comment:
As per other components, instead of doing this check we should use the standard method in the app i.e. create a getter called ```invalid``` and use this value here

e.g.
```
 get invalid(): boolean {
    return !this.formControl.valid && this.formControl.dirty;
  }
```
Then the HTML becomes 
```[class.invalid-bar]="invalid"```